### PR TITLE
Adds sanity to Xenoarch's replicator artifact

### LINKED ISSUE #7195: Ghosts can use alien machines, resulting in everyone killing each other.
Im finding that Alien Machines are able to be used by ghosts, making shit like viscerators and leading to mass mayhem and shitshows.

I dont know if this was intentional or not but this HAS to stop and I'm probably going to be deleting every one I see until this is changed for the sake of the round that they show up in.

### PR DESCRIPTION
**Pull request by jknpj**
Fucking faggot ghosts man i swear to god

Closes #7195

### CODE REVIEW

File: code/modules/research/xenoarchaeology/artifact/artifact_replicator.dm [hunk 1]
```diff
@@ -125,6 +125,11 @@
 	interact(user)
 
 /obj/machinery/replicator/interact(mob/user)
+	if(isobserver(user))
+	to_chat(user, "<span class='rose'>Your ghostly hand goes right through!</span>")
```
Comment (ComicIronic) [inline code/modules/research/xenoarchaeology/artifact/artifact_replicator.dm]: Indent.

File: code/modules/research/xenoarchaeology/artifact/artifact_replicator.dm [hunk 2]
```diff
@@ -125,6 +125,11 @@
...
+			return
```
Comment (ComicIronic) [inline code/modules/research/xenoarchaeology/artifact/artifact_replicator.dm]: TOO MUCH INDENT

File: code/modules/research/xenoarchaeology/artifact/artifact_replicator.dm [hunk 3]
```diff
@@ -125,6 +125,11 @@
...
+	if(user.incapacitated())
+			return
```
Comment (ComicIronic) [inline code/modules/research/xenoarchaeology/artifact/artifact_replicator.dm]: Too much indent.

**Review by ComicIronic — CHANGES_REQUESTED** [review]
Bad

**Comment by Probe1** [comment]
We intentionally did not add sanity to it.  If people want it now that's fine I suppose.

**Comment by Sprok0** [comment]
This was the best part of that artifact what are you doing

**Comment by ghost** [comment]
Keep it xenoarchy by it having a chance to not care about ghost hands

**Comment by ghost** [comment]
Or hell, if ghosts are really bothering you, do it the IC way and get a chaplain to bless that shit with holy water

**Comment by jknpj** [comment]
@Sprok0 & @Probe1
I know it is the iconic part of it but it is way too much exploitable and allowing ghosts to summon hostile mobs is just bananas

@Skullyton 
>deal with it ICly

Kinda hard when there's about fifty mimics and dozen viscerators around it because ghosts

**Comment by Sprok0** [comment]
At least make it hit its own buttons sometimes if you're that set on blocking ghosts from it

**Comment by Shadowmech88** [comment]
If the volume of mobs is a problem, add a cooldown to the artifact.
Removing the ability for ghosts to use it is dumb. Chaplains exist, the problem could be solved IC if people in the round cared enough.

**Comment by 9600bauds** [comment]
EVERYTHING ghost-related could be solved IC via chaplain
That's not really a compelling argument to let ghosts fuck with important things, which is why we normally don't let ghosts fuck with important things

**Comment by Shadowmech88** [comment]
Alright but in this case letting ghosts fuck with it was explicitly left in as a feature.

**Comment by ComicIronic** [comment]
Does `incapacitated()` stop ghosts?

**Comment by 9600bauds** [comment]
ye

**Comment by jknpj** [comment]
Yes but i kinda wanted a ghostly flavor text so i ended using `isobserver` for them and `incapacitated()` for the living ones

**Review by ComicIronic — APPROVED** [review]

**Comment by jknpj** [comment]
Will do something about this without removing the FEAAAATUUUUREEEEEEEEE

**Pull request closed without merging**